.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on:
+  - push
+  - pull_request
```

Comment:
If you don't like building PR twice, you can use something like

```yaml
on:
  push:
    branches:
      - master
    tags: '*'
  pull_request:
```

to avoid triggering the build on push to non-`master` branches.

Reply:
Thanks, I was thinking this might be excessive (it was previously turned off in the Travis config)